src/components/CreateVideo.js

```diff
@@ -100,19 +167,23 @@ const CreateVideo = (props) => {
       artist: results[songItem].artist.name,
       deezer_id: results[songItem].id,
       video_title: videoTitle.title,
+      im_group: videoParams.im_group,
+      jitter: videoParams.jitter,
+      depth: videoParams.depth,
+      truncation: videoParams.truncation,
+      pitch_sensitivity: videoParams.pitch_sensitivity,
+      tempo_sensitivity: videoParams.tempo_sensitivity,
+      smooth_factor: videoParams.smooth_factor,
```

Comment:
This looks like it solves that issue where videoParams was being shoehorned into the POST.  This makes much more sense.  Thank you for catching and solving this.